Framework/Core/src/RootConfigParamHelpers.cxx

```diff
@@ -158,7 +190,7 @@ void ptreeToMember(boost::property_tree::ptree const& value,
 }
 
 // Convert a DataMember to a ConfigParamSpec
-ConfigParamSpec memberToConfigParamSpec(const char* tname, TDataMember const* dm, void* ptr)
+ConfigParamSpec memberToConfigParamSpec(const char* tname, TDataMember* dm, void* ptr)
```

Comment:
Why not const anymore?

Reply:
ROOT's check for STL container somehow is not marked const and thus will not work with const pointer. I guess this should be reported as a bug.